apps/api/src/graphql/schema/programs.ts

```diff
@@ -66,10 +66,22 @@ input ProgramRequirementsQuery {
     programId: String!
 }
 
+input majorsQuery {
```

Comment:
Very silly of me to not tell you this earlier but types in GraphQL are in `PascalCase`, so `MajorsQuery`